src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -942,12 +941,12 @@ bool VM_Version::glibc_env_set(char *disable_str) {
   char *env_val = disable_str;
   const char *env = getenv(TUNABLES_NAME);
   if (env && strcmp(env, env_val) == 0) {
-#if !INCLUDE_CPU_FEATURE_ACTIVE && !INCLUDE_LD_SO_LIST_DIAGNOSTICS
-    if (ShowCPUFeatures) {
-      tty->print_cr("Environment variable already set, both glibc CPU_FEATURE_ACTIVE and ld.so --list-diagnostics are unavailable - re-exec suppressed: " TUNABLES_NAME "=%s", env);
+    if (!INCLUDE_CPU_FEATURE_ACTIVE && !INCLUDE_LD_SO_LIST_DIAGNOSTICS) {
```

Comment:
Have you tested compilation with these disabled? Looks to me like the compiler could complain about unreachable code when the condition is always false...

Reply:
I have tried all the 3 compilation cases only now. I do not see such compiler warnings (although different compilations and compilers produce different warnings) but I could fix some other problems. Which will make me update #112.